schematic/models/validate_attribute.py

```diff
@@ -9,6 +9,7 @@
 
 import numpy as np
 import pandas as pd
+from pandas._libs.missing import NAType
```

Comment:
When _ is used as in _libs that's generally understood that it's nto safe to use outside its context, in this case pandas. In schematic DB I use pandas.isnull(value) to test if its a missing value.